File: python/download.py
import os

import synapseclient 
import synapseutils 

username = os.getenv("SYNAPSE_USERNAME")
password = os.getenv("SYNAPSE_PASSWORD")
project_id = os.getenv("SYNAPSE_PROJECT_ID")

syn = synapseclient.Synapse() 
syn.login(username, password) 
files = synapseutils.syncFromSynapse(syn, project_id) 

File: R/download.R
library(synapser)
library(synapserutils)

username <- Sys.getenv("SYNAPSE_USERNAME")
password <- Sys.getenv("SYNAPSE_PASSWORD")
project.id <- Sys.getenv("SYNAPSE_PROJECT_ID")

synLogin(username, password) 
files <- synapserutils::syncFromSynapse(project.id) 
